### PR TITLE
testing deploying from diff branches

### PR DESCRIPTION
**Pull request by zachmcclurg-ccpo**
PR to start deploying cdk using a context obj to pass the branch repo so we can differentiate between dev and main.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
           npm ci
       - name: Synthesize CDK template - Sandbox
         run: |
-          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true"
+          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true" -c "atat:VersionControlBranch=repoBranch"
```
Comment (jasonburkert-ccpo) [inline .github/workflows/test.yml]: Make the default value of the context key `develop` so it works similarly by default to the way it did before.

```suggestion
          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true" -c "atat:VersionControlBranch=develop"
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -56,6 +56,7 @@ jobs:
           -c "atat:ApiDomainName=ci.atat.fake.test"
           -c "atat:ApiCertificateArn=arn:aws:us-east-1:123456789012:certificate/aec3ac93-692e-406e-afef-642fb31afc41"
           -c "atat:VpcFlowLogBucket=arn:aws:us-east-1:s3::123456789012:flow-logs-123456789012-us-east-1" 
+          -c "atat:VersionControlBranch=repoBranch"
```
Comment (jasonburkert-ccpo) [inline .github/workflows/test.yml]: Similarly, make this `develop` too

**Review by jasonburkert-ccpo — CHANGES_REQUESTED** [review]
A small change requested.

File: .github/workflows/test.yml (hunk 1 above)
Comment (zachmcclurg-ccpo) [reply .github/workflows/test.yml]: done

**Comment by jasonburkert-ccpo** [comment]
> A small change requested. This value will be more meaningful to a git user.

Fixed in https://github.com/dod-ccpo/atat-web-api/pull/1570/commits/6e3c1d3cd18dc7d4f60797f38c69d761932b41ac

**Review by jasonburkert-ccpo — APPROVED** [review]

**Pull request merged**